RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 17.2
 -----
 
+* [*] Fixed an issue where post author display names weren't visible. [#16297]
```

Comment:
It might be worth specifying that this was a bug specific to self-hosted sites (which I understand from the issue was the case).
(The convention of adding `[#16297]` to the release notes is an aid to internal beta testers. When editorial make the App Store release notes from these bullet points, they won't be looking at the PR so won't know which site types were affected by this bug.)

Reply:
Good call, and thanks for the extra information ! I'll get this fixed.

Reply:
Sure, that's not documented anywhere (I'd like to add a "how to maintain release notes" section to our release notes) 🙇

Reply:
> I'd like to add a "how to maintain release notes" section to our release notes

That's something I'll follow up on myself.

Reply:
This is updated. 👍